datasources/src/main/java/org/opensearch/sql/datasources/transport/TransportUpdateDataSourceAction.java

```diff
@@ -55,9 +58,15 @@ protected void doExecute(
       ActionListener<UpdateDataSourceActionResponse> actionListener) {
     try {
       dataSourceService.updateDataSource(request.getDataSourceMetadata());
+      String responseContent =
+              new JsonResponseFormatter<String>(PRETTY) {
+                @Override
+                protected Object buildJsonObject(String response) {
+                  return response;
+                }
+              }.format("Updated DataSource with name " +  request.getDataSourceMetadata().getName());
       actionListener.onResponse(
-          new UpdateDataSourceActionResponse(
-              "Updated DataSource with name " + request.getDataSourceMetadata().getName()));
+          new UpdateDataSourceActionResponse(responseContent));
```

Comment:
Delete does not need this change since there is no body - since it is using HTTP 204 response. Do we want to change that and return a message similar to POST and PUT?